extension/lib/respository/BigcommerceCustomerRepository.js

```diff
@@ -0,0 +1,29 @@
+class BigcommerceCustomerRepository {
```

Comment:
There is already same root under different path, please put it on the same place ./extension/lib/bigcommerce/CustomerRepository

Reply:
done

Reply:
I guess we can talk about this and come to a common ground when we're both here.

Reply:
done